cmd/utility.go

```diff
@@ -139,3 +139,16 @@ func removeKeys(aeacusPath string) {
 		failPrint("Failed to stat " + keyFile)
 	}
 }
+
+func lines2map(lines string) map[string]string {
```

Comment:
Just for reference, it's uncommon to see this type of naming convention in go programs. Typically, people use `MixedCaps` for exported names and `mixedCaps` for not. Check out [Effective Go](https://go.dev/doc/effective_go) for more general info.